index.html

```diff
@@ -16,12 +16,100 @@
+        <section class="grid-container2">
+            <section class="grid-item1 box7">HTML/CSS</section>
+            <section class="grid-item1 box8">Ruby on Rails</section>
+            <section class="grid-item1 box9">Javascript</section>
+        </section>
+        <section class="rectangle6">
+            <h4 class="text text5">See this project</h4>
+            <img class="img img3" src="arrowsymbol.svg" alt="arrow symbol for webpage">
+        </section>
+        <section class="projectpageimage4"></section>
+        <section class="rectangle7">
+            <h2 class="text text4">Project name goes here</h2>
+        </section>
+        <section class="grid-container2">
+            <section class="grid-item1 box10">HTML/CSS</section>
+            <section class="grid-item1 box11">Ruby on Rails</section>
+            <section class="grid-item1 box12">Javascript</section>
+        </section>
+        <section class="rectangle8">
+            <h4 class="text text5">See this project</h4>
+            <img class="img img3" src="arrowsymbol.svg" alt="arrow symbol for webpage">
+        </section>
+
+        <section class="projectpageimage5"></section>
+        <section class="rectangle9">
+            <h2 class="text text4">Project name goes here</h2>
+        </section>
+        <section class="grid-container2">
+            <section class="grid-item1 box13">HTML/CSS</section>
+            <section class="grid-item1 box14">Ruby on Rails</section>
+            <section class="grid-item1 box15">Javascript</section>
+        </section>
+        <section class="rectangle10">
+            <h4 class="text text5">See this project</h4>
+            <img class="img img3" src="arrowsymbol.svg" alt="arrow symbol for webpage">
+        </section>
 
+        <section class="projectpageimage6"></section>
+        <section class="rectangle11">
+            <h2 class="text text4">Project name goes here</h2>
+        </section>
+        <section class="grid-container2">
+            <section class="grid-item1 box16">HTML/CSS</section>
+            <section class="grid-item1 box17">Ruby on Rails</section>
+            <section class="grid-item1 box18">Javascript</section>
+        </section>
+        <section class="rectangle12">
+            <h4 class="text text5">See this project</h4>
+            <img class="img img3" src="arrowsymbol.svg" alt="arrow symbol for webpage">
+        </section>
+        <section class="About_section">
+            <div class="text abouttext">About</div>
+            <p class="text aboutpara">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+        </section>
+        <section class="rectangle13">
+            <h6 class="text getmycv">Get my CV</h6>
+            <img class="img img4" src="ic_download.svg" alt="download icon">
+        </section>
```

Comment:
- [ ] As per Figma design there is about section which you have not completely implemented. Please fixes this issue.